src/react.test.tsx

```diff
@@ -87,6 +88,26 @@ it("should update state (options configuration)", () => {
   expect(result.current).toBe(1);
 });
 
+
+it("should update state by subscribing to multiple events", () => {
+  const positiveNumberEvent = createEventDefinition<number>()("positive");
+  const negativeNumberEvent = createEventDefinition<number>()("negative");
+
+  const { result } = renderHook(() =>
+    useBusState.configure( {
+      subscriber: stateSubscribeDefinition(["positive", "negative"])})(0), {
```

Comment:
This kind of reads pretty cryptically for me. Maybe the name of the function might be better?